### PR TITLE
fixed example

### PR DESCRIPTION
**Pull request by pinpong**
## Summary
Some parts of the example don't work.

**Fixed issue**: 

1. fixed CSE
2. fixed settings
3. fixed styles
4. show all available payment methods

### CODE REVIEW

**Comment by descorp** [comment]
Hey @pinpong 

This is super helpfull! Thanks a lot :)

Could you also attach screenshots for `Checkout` and `CSE` pages?

**Comment by pinpong** [comment]
Screenshot_1682499019
Screenshot_1682499013

File: example/src/App.js [hunk 1]
```diff
@@ -25,6 +25,11 @@ import { DEFAULT_CONFIGURATION } from './Configuration';
 
 const Stack = createNativeStackNavigator();
 
+const SettingsButton = ({ navigation }) => {
+  return (
+    <Button onPress={() => navigation.navigate('Settings')} title="Edit" />
+  );
+};
```
Comment (descorp) [inline example/src/App.js:32]: This is nice :)

File: example/src/Views/SettingsView.js [hunk 2]
```diff
@@ -104,9 +112,11 @@ const SettingFormView = ({ navigation: { goBack } }) => {
       <FormTextInput
         title="Shopper Reference"
         value={shopperReference}
-        onChangeText={setShopperLocale}
+        onChangeText={setShopperReference}
```
Comment (descorp) [inline example/src/Views/SettingsView.js:115]: Thanks 😅

**Review by descorp — DISMISSED** [review]

**Comment by descorp** [comment]
@pinpong 

Let me know if this is it - I'll merge the PR

**Comment by pinpong** [comment]
You can merge the changes. I'm done here. Working on a idea to fix the dialog issue.

File: example/src/App.js [hunk 3]
```diff
@@ -42,18 +47,20 @@ const App = () => {
...
             options={({ navigation }) => ({
-              title: 'Adyen React Native',
-              headerRight: () => (
-                <Button
-                  onPress={() => navigation.navigate('Settings')}
-                  title="Edit"
-                />
-              ),
+              title: 'Checkout',
+              headerRight: () => <SettingsButton navigation={navigation} />,
```
Comment (descorp) [inline example/src/App.js]: Sonar cloud is complaining about this
https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=195&id=Adyen_adyen-react-native&open=AYe8wPYFweZLuJAQoljM

```suggestion
              headerRight: () => (<SettingsButton navigation={navigation} />),
```

**Review by descorp — APPROVED** [review]

**Comment by descorp** [comment]
Thanks @pinpong

**Pull request merged**